packages/simulator/streams/kommuner.js

```diff
@@ -24,12 +17,32 @@ const data = require('../data/kommuner.json')
 const population = require('./population')
 const packageVolumes = require('./packageVolumes')
 const postombud = require('./postombud')
+
 const inside = require('point-in-polygon')
 const commercialAreas = from(require('../data/scb_companyAreas.json').features)
 const { generateBookingsInKommun } = require('../simulator/bookings')
 const bookingsCache = require('../streams/cacheBookingStream')
 const Pelias = require('../lib/pelias')
 const { generatePassengers } = require('../simulator/passengers')
+let kommunNames = [
```

Comment:
I moved this to [setup.js in my just merged PR](https://github.com/PredictiveMovement/digital-twin/pull/129/files#diff-e2e35092d57a9444ecc82b12e9a0278d8cda6cfacccb0ca90c374e41c9f55d3d) and was thinking ahead of how it might be easier to set it up separately per project in that way.

Reply:
Maybe something like this would be better to keep configuration separate from the code and be able to check and diff config files between each other as they grow:

https://gist.github.com/StefanWallin/27d3d501e5fc0ff4fe0a733f10ebc6a0